Fix:

error: assignment to ‘char **’ from incompatible pointer type ‘char *’ [-Wincompatible-pointer-types]
error: assignment to ‘int *’ from ‘int’ makes pointer from integer without a cast [-Wint-conversion]

https://bugs.ghostscript.com/show_bug.cgi?id=707649
https://git.ghostscript.com/?p=ghostpdl.git;a=commitdiff;h=90cabe08422afdd16bac5dd9217602679d943045
--- pdf/pdf_sec.c.orig	2024-03-06 10:52:27
+++ pdf/pdf_sec.c	2024-03-14 21:16:11
@@ -183,8 +183,8 @@
          * this easy: the errors we want to ignore are the ones with
          * codes less than 100. */
         if ((int)err < 100) {
-            NewPassword = Password;
-            NewLen = Len;
+            *NewPassword = Password;
+            *NewLen = Len;
             return 0;
         }
 
